cmd/triggers/create.go

```diff
@@ -305,7 +305,7 @@ func triggerRequestFromFlags(flags createCmdFlags, isDev bool) api.TriggerReques
 	return req
 }
 
-func triggerRequestViaHook(clients *shared.ClientFactory, path string, isDev bool) (api.TriggerRequest, error) {
+func triggerRequestViaHook(ctx context.Context, clients *shared.ClientFactory, path string, isDev bool) (api.TriggerRequest, error) {
```

Comment:
note: Example of the cascading changes to pass `ctx` into areas where `.Execute` is called.

Reply:
@mwbrooks Legend ⭐

These changes seem tedious, but I am so hopeful that future changes can make use of the real `ctx` without hassle now.